src/services/recim/series.ts

```diff
@@ -0,0 +1,59 @@
+import http from '../http';
+import { Match } from './match';
+import { TeamRecord } from './team';
+
+export type Series = {
+  ID: number;
+  Name: string;
+  StartDate: string;
+  EndDate: string;
+  ActivityID: number;
+  Type: string;
+  Status: string;
+  Match: Match[];
+  TeamStanding: TeamRecord[];
+};
+
+type CreatedSeries = {
+  ID: number;
+  Name: string;
+  StartDate: string;
+  EndDate: string;
+  ActivityID: number;
+  TypeID: number;
+  StatusID: number;
+};
+
+type UploadSeries = {
+  Name: string;
+  StartDate: string;
+  EndDate: string;
+  ActivityID: number;
+  TypeID: number;
+  NumberOfTeamsAdmitted: number; //used for subsequent series creation post initial setup
+};
+
+type PatchSeries = {
+  Name: string;
+  StartDate: string;
+  EndDate: string;
+  StatusID: number;
+};
+
+//Series Routes
+const createSeries = async (
+  referenceSeriesID: number,
+  newSeries: UploadSeries,
+): Promise<CreatedSeries> => {
+  var subQuery = referenceSeriesID === null ? '' : `?referenceSeriesID=${referenceSeriesID}`;
+  return await http.post(`recim/series${subQuery}`, newSeries);
```

Comment:
Any data that is part of the object to be created should be in the post body. Could you clarify exactly what referenceSeriesID is supposed to do here?

If the purpose is to create a series using another existing one as a template, I would recommend creating a different API route & UI service method for that.

Even that recommendation would not be considered truly "restful" as operations should perform one action (create, retrieve, update, or destroy). When you create a copy of another resource, I would think that the most responsible way to do so would be to first retrieve the series, remove the ID and update as necessary, then pass the updated object to the POST route.